src/js/angular/export/controllers.js

```diff
@@ -153,13 +153,13 @@ exportCtrl.controller('ExportCtrl',
              * @param {Object} graphsByValue
              * @param {Object} JSONLDMode (name and mode link)
              */
-            function downloadJSONLDExport(format, context, link, forSelectedGraphs, repo, graphsByValue, JSONLDMode) {
```

Comment:
Can you just update the jsdoc to account changes in the parameters list. Do this for all methods you changed